src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.js

```diff
@@ -274,4 +286,12 @@ export default compose(
             key: ONYXKEYS.IS_LOADING_REPORT_DATA,
```

Comment:
Looks like there is two `withWindowDimensions` is used, one should be removed.
Coming from - https://expensify.slack.com/archives/C01GTK53T8Q/p1688970802542999